### PR TITLE
[plugin.video.s04tv] 3.1.3

### PR DESCRIPTION
**Pull request by maloep**
update due to website changes

### Checklist:

- [ X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [ X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X ] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.s04tv/default.py [hunk 1]
```diff
@@ -12,18 +14,25 @@
...
 import urllib, urllib2
 from urlparse import *
-import xml.etree.ElementTree as ET
+#import xml.etree.ElementTree as ET
+import cookielib
+try:
+# Python 2.6-2.7 
+    from HTMLParser import HTMLParser
+except ImportError:
+    # Python 3
```
Comment (romanvm) [inline plugin.video.s04tv/default.py]: If you want to support Python 3, you need more than that 😄

File: plugin.video.s04tv/default.py [hunk 2]
```diff
@@ -34,418 +43,160 @@
 sys.path.append( os.path.join( BASE_RESOURCE_PATH, "lib" ) )
 sys.path.append( os.path.join( BASE_RESOURCE_PATH, "lib", "BeautifulSoup" ) )
 
-from BeautifulSoup import BeautifulSoup
-from BeautifulSoup import Tag
-import mechanize
 
 __language__ = __addon__.getLocalizedString
```
Comment (romanvm) [inline plugin.video.s04tv/default.py]: Using names with __double_underscores__ is not recommended because they are reserved for internal Python names.

Comment (maloep) [reply plugin.video.s04tv/default.py]: Ok, thanks. I already changed some issues that codacy reported and will fix the things that you named as well.

**Comment by Lunatixz** [comment]
@romanvm LGTM

**Pull request merged**